frontend/chat-plugin/src/airyRenderProps/AiryInputBar/index.tsx

```diff
@@ -4,10 +4,14 @@ import style from './index.module.scss';
 
 type Props = {
   sendMessage: (text: string) => void;
+  cyButtonId?: string;
+  cyTextareaId?: string;
 };
 
 const AiryInputBar = (props: Props) => {
   const [messageString, setMessageString] = useState('');
+  const cyButtonId = props.cyButtonId;
```

Comment:
Since these are global variables, I would not pass them via props, but require them right here

Reply:
fair